.travis.yml

```diff
@@ -1,19 +1,10 @@
-language: erlang
+language: elixir
+elixir:
+  - 0.15.1
```

Comment:
Will do it later today
On Sat, 16 May 2015 at 10:28 am Lau Taarnskov <notifications@github.com>
wrote:

>  [image: Boxbe] <https://www.boxbe.com/overview> This message is eligible
> for Automatic Cleanup! (notifications@github.com) Add cleanup rule
> <https://www.boxbe.com/popup?url=https%3A%2F%2Fwww.boxbe.com%2Fcleanup%3Ftoken%3DIvK9S%252BU3Wf82H3oaL2SgZLg%252BUIUx9vzKUy%252Bz%252BBHJ6w0ad5t1WsXG%252F7inFd2ano0Ih%252BcnEKkr2xnelMUPkRt5nvRpRW5V5zx8S5xXBHc8o%252BMFvYGtQirZc4rgoe%252FBaZdcFO7OMeBWfZd2jVzThtIjmQ%253D%253D%26key%3DIXVYSEwlb5UN%252F7wBir%252FDJc9QK9GoWQv6wcMiJ%252FIuQEI%253D&tc_serial=21344173961&tc_rand=158412182&utm_source=stf&utm_medium=email&utm_campaign=ANNO_CLEANUP_ADD&utm_content=001>
> | More info
> <http://blog.boxbe.com/general/boxbe-automatic-cleanup?tc_serial=21344173961&tc_rand=158412182&utm_source=stf&utm_medium=email&utm_campaign=ANNO_CLEANUP_ADD&utm_content=001>
>
> In .travis.yml
> <https://github.com/lau/kalends/pull/3#discussion_r30457499>:
>
> > @@ -1,19 +1,10 @@
> > -language: erlang
> > +language: elixir
> > +elixir:
> > +  - 0.15.1
>
> This doesn't seem to work in travis. But support for 0.15.1 is not really
> needed anymore. Can you just delete this line? Then I'll merge it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lau/kalends/pull/3/files#r30457499>.
>